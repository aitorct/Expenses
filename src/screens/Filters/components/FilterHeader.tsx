import React from 'react';
import {StyleSheet, Text} from 'react-native';
import fontSizes from '@theme/fontSizes';
import sizes from '@theme/sizes';
import colors from '@theme/colors';

const FilterHeader = ({title}: {title: string}) => (
  <Text style={styles.title}>{title}</Text>
);

const styles = StyleSheet.create({
  title: {
    fontSize: fontSizes.subtitle,
    fontWeight: '500',
    paddingBottom: sizes.s12,
    color: colors.text,
  },
});

export default FilterHeader;
