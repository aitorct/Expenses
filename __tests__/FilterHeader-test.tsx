import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import FilterHeader from '@screens/Filters/components/FilterHeader';

it('FilterHeader renders correctly', () => {
  const tree = renderer.create(<FilterHeader title={'Title'} />).toJSON();
  expect(tree).toMatchSnapshot();
});
