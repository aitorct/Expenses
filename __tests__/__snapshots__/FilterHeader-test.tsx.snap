// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterHeader renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#000000",
      "fontSize": 20,
      "fontWeight": "500",
      "paddingBottom": 12,
    }
  }
>
  Title
</Text>
`;
